### PR TITLE
@ #122 | kubernestes-stack v0.1.0 release announcement

### LINKED ISSUE #122: should create "kubernestes-stack v0.1.0 release announcement" blog post
we're going to release kubernestes-stack-cookbook v0.1.0 soon, we need a blog post for this.

### PR DESCRIPTION
**Pull request by hoatle**
connect #122

### CODE REVIEW

File: source/_posts/2017-08-26-kubernetes-stack-v0-1-0-release.markdown [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+It's been more than two months since we introduced the inception of the `kubernetes-stack` cookbook
+from [this blog post](2017/06/15/how-to-extend-teracy-dev-to-work-with-kubernetes/) and today we're
+very happy to announce the very first milestone release of it.
+
+<!-- more -->
+
+## What's new?
+
+This milestone release helps us to manage the installation of these packages: `kubectl`, `heml`,
```
Comment (hieptranquoc) [inline source/_posts/2017-08-26-kubernetes-stack-v0-1-0-release.markdown]: @hoatle `helm`

Comment (hoatle) [reply source/_posts/2017-08-26-kubernetes-stack-v0-1-0-release.markdown]: fixed, thank you.

File: source/_posts/2017-08-26-kubernetes-stack-v0-1-0-release.markdown [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    - devops
+cover: 
+description: kubernetes-stack cookbook v0.1.0 release announcement
+keywords: kubernetes, kubernetes-stack, gcloud, k8s, utilities, devops
+published: true
+---
+
+It's been more than two months since we introduced the inception of the `kubernetes-stack` cookbook
+from [this blog post](2017/06/15/how-to-extend-teracy-dev-to-work-with-kubernetes/) and today we're
+very happy to announce the very first milestone release of it.
```
Comment (hoavt) [inline source/_posts/2017-08-26-kubernetes-stack-v0-1-0-release.markdown]: the very first milestone release of it. --> its very first milestone release.

File: source/_posts/2017-08-26-kubernetes-stack-v0-1-0-release.markdown [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+You can install, remove any valid versions of those packages onto Ubuntu or CentOS platforms and more
+supported platforms will come.
+
+
+## What's next?
+
+This cookbook is designed to provide utilities for us to work with `Kubernetes` stack, so the next
+milestone will introduce `kubernetes` resource to set up and install `Kubernetes` server which
+should work the same as `Minukube` with better developer productivity and enhancement. This
```
Comment (hoavt) [inline source/_posts/2017-08-26-kubernetes-stack-v0-1-0-release.markdown]: better developer productivity and enhancement --> better developer productivity enhancement

File: source/_posts/2017-08-26-kubernetes-stack-v0-1-0-release.markdown [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+will allow our developers to have a quick local deployment test with k8s on their workstations.
+
+By using this, we can save a lot of server cost, reduce testing time and speed up delivery time of
+application deployments.
```
Comment (hoavt) [inline source/_posts/2017-08-26-kubernetes-stack-v0-1-0-release.markdown]: deployments --> deployment.

File: source/_posts/2017-08-26-kubernetes-stack-v0-1-0-release.markdown [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+Developers will understand `Kubernetes` more deeply when they can do whatever they want with it,
```
Comment (hoavt) [inline source/_posts/2017-08-26-kubernetes-stack-v0-1-0-release.markdown]: --> they want with it.  It is very safe! They can destroy...

File: source/_posts/2017-08-26-kubernetes-stack-v0-1-0-release.markdown [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+it's very safe, they can destroy and set up a new deployment anytime they want without causing any
+harm to their co-workers.
+
+You should checkout how it should be used with `teracy-dev` here at: https://github.com/acme101/dev-setup/blob/develop/chef/main-cookbooks/acme/recipes/k8s.rb
```
Comment (hoavt) [inline source/_posts/2017-08-26-kubernetes-stack-v0-1-0-release.markdown]: --> Checkout how it should be used with `teracy-dev` at...

**Comment by hoatle** [comment]
@hoavt PR updated.

**Comment by hoatle** [comment]
@hieptranquoc FYI, you can review the built Docker image here https://hub.docker.com/r/hoatle/teracy-blog/tags/ (hoatle/teracy-blog:tasks-122-kubernestes-stack-v0.1.0-release) by following https://github.com/teracyhq/blog/blob/master/README.markdown#how-to-review-others-work-and-prs-pull-requests

**Comment by hoavt** [comment]
It's Ok to merge.

**Pull request merged**